virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -160,6 +160,16 @@ def create_disk(
         )
 
 
+def _all_disk_targets():
+    devs = []
+    vmx = vm_xml.VMXML.new_from_dumpxml(vm_name)
+    for disk in (vmx.get_devices("disk") or []):
+        tgt = (disk.target or {}).get("dev")
```

Comment:
Change `tgt` to a meaningful word instead of abbreviation. For example, `target_dev`